sdk-python/littlehorse/utils.py

```diff
@@ -0,0 +1,167 @@
+    VariableType.JSON_ARR: list[Any],
+    VariableType.DOUBLE: float,
+    VariableType.BOOL: bool,
+    VariableType.STR: str,
+    VariableType.INT: int,
+    VariableType.BYTES: bytes,
+}
+
+TYPE_VARIABLE_MAP = {value: key for key, value in VARIABLE_TYPE_MAP.items()}
+
+
+def timestamp_now() -> Timestamp:
+    """Return a Timestamp protobuf object.
+
+    Returns:
+        Timestamp: Timestamp protobuf object.
+    """
+    current_time = Timestamp()
+    current_time.GetCurrentTime()
+    return current_time
+
+
+def parse_value(value: Any) -> VariableValue:
+    """Receives a python variable and return a VariableValue.
+
+    Args:
+        value (Any): Any value returned by a method.
+
+    Returns:
+        VariableValue: LH Variable.
+    """
+
+    def json_encoder(value: Any) -> Any:
+        if hasattr(value, "__dict__"):
+            return vars(value)
+        return value
+
+    if value is None:
+        return VariableValue(type=VariableType.NULL)
+    if isinstance(value, bool):
+        return VariableValue(type=VariableType.BOOL, bool=value)
+    if isinstance(value, str):
+        return VariableValue(type=VariableType.STR, str=value)
+    if isinstance(value, int):
+        return VariableValue(type=VariableType.INT, int=value)
+    if isinstance(value, float):
+        return VariableValue(type=VariableType.DOUBLE, double=value)
+    if isinstance(value, bytes):
+        return VariableValue(type=VariableType.BYTES, bytes=value)
+    if isinstance(value, dict):
+        return VariableValue(
+            type=VariableType.JSON_OBJ,
+            json_obj=json.dumps(value, default=json_encoder),
+        )
+    if isinstance(value, list):
+        return VariableValue(
+            type=VariableType.JSON_ARR,
+            json_arr=json.dumps(value, default=json_encoder),
+        )
+
+    return VariableValue(
+        type=VariableType.JSON_OBJ, json_obj=json.dumps(value, default=json_encoder)
+    )
+
+
+def parse_type(lh_type: VariableType) -> Any:
+    """Receives a LH type and return a python type.
+
+    Args:
+        lh_type (VariableType): LH Type.
+
+    Returns:
+        Any: Python type.
+    """
+    return VARIABLE_TYPE_MAP[lh_type]
+
+
+def extract_value(lh_value: VariableValue) -> Any:
+    """Receives a LH value and maps it to a python object.
+
+    Args:
+        lh_value (VariableValue): LH Value.
+
+    Returns:
+        Any: Python value.
+    """
+    if lh_value.type == VariableType.STR:
+        return lh_value.str
+    if lh_value.type == VariableType.INT:
+        return lh_value.int
+    if lh_value.type == VariableType.DOUBLE:
+        return lh_value.double
+    if lh_value.type == VariableType.BYTES:
+        return lh_value.bytes
+    if lh_value.type == VariableType.BOOL:
+        return lh_value.bool
+    if lh_value.type == VariableType.JSON_OBJ:
+        return json.loads(lh_value.json_obj)
+    if lh_value.type == VariableType.JSON_ARR:
+        return json.loads(lh_value.json_arr)
```

Comment:
Same comment—this can fail, so do we want to throw a custom LH exception here or leave it as-is? Up to you.